src/boot/z_std_dma.c

```diff
@@ -197,7 +265,13 @@ void DmaMgr_Error(DmaRequest* req, const char* file, const char* errorName, cons
     Fault_AddHungupAndCrashImpl(buff1, buff2);
 }
 
-const char* DmaMgr_GetFileNameImpl(uintptr_t vrom) {
+/**
+ * Searches the filesystem for the entry containing the address `vrom`.
+ *
+ * @param vrom vrom address
```

Comment:
```suggestion
 * @param vrom virtual rom address
```
Might as well expand the acronym a bit for clarity rather than be repetitive.